### ISSUE TITLE
Training dataset specific encoders

### ISSUE THREAD

**Issue by demiahmed** [EzioBy/padinv#6]
Hi I noticed that there are pre-trained encoder models for specific datasets like churches, faces etc.

How do I go about training my own encoder that's specific to my own dataset? Say a dataset of website landing pages, or keyboards? I already have a pre-trained StyleGAN2 generator model for my custom dataset.

Do I need to follow a specific methodology?

Any advice will be highly appreciated

**Comment by EzioBy** [comment]
Since you have got a pre-trained StyleGAN2 generator model for the customized dataset, I think you can directly adopt our codes for training encoders on LSUN Church or LSUN Bedroom (so the identity-matching loss will not be included). Note that you need to reset the dataset path and pre-trained GAN path based on your setting in **encoder_stylegan2_*_train.sh**. I also think you may need to rebalance the loss weights.
Hope these will be of some help.
